shared/src/store/action-reducers/alarm-group.ts

```diff
@@ -10,12 +10,13 @@ import { AlarmGroup } from '../../models/alarm-group';
 import { AlarmGroupVehicle } from '../../models/utils/alarm-group-vehicle';
 import type { Mutable } from '../../utils';
 import { cloneDeepMutable, UUID, uuidValidationOptions } from '../../utils';
+import { IsStringLiteralUnion } from '../../utils/validators';
 import type { Action, ActionReducer } from '../action-reducer';
 import { ReducerError } from '../reducer-error';
 import { getElement } from './utils/get-element';
 
 export class AddAlarmGroupAction implements Action {
-    @IsString()
```

Comment:
There is a ts error if the two strings are not identical. 

In the future, the boilerplate could maybe be reduced via a `@IsConstant` decorator that creates n instance and looks at what the properties value is by default.

Reply:
I've added a `IsValue` for now.